.github/workflows/ci.yml

```diff
@@ -231,7 +227,7 @@ jobs:
       - "publish_pypi"
     runs-on: "ubuntu-20.04"
     env:
-      SLACK_WEBHOOK_URL: "${{ secrets.SLACK_WEBHOOK_URL }}"
+      SLACK_WEBHOOK_URL: "${{ '{{ secrets.SLACK_WEBHOOK_URL }}' }}"
```

Comment:
Was this intentional?

Reply:
Yes, this matches the LTM-1.6 from Golden Config.

Reply:
Is this a syntax you're familiar with? I'm not sure I understand how this works.

Reply:
I reverted it, I think the source I got this from might be incorrect.